examples/react-cra/src/components/Navigation.tsx

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
 import { NavLink } from 'react-router-dom';
 
+import { Search } from './Search';
+
 export const Navigation = () => {
+  const queryClient = new QueryClient();
+
   return (
     <nav className="topnav">
       <NavLink to="/stoplight-project">Stoplight Project</NavLink>
       <NavLink to="/zoom-api">Zoom API</NavLink>
+      <QueryClientProvider client={queryClient}>
```

Comment:
My goal is to not have to include this here because I have already included it in the component file [here](https://github.com/stoplightio/elements/pull/1749/files#diff-8eac8a4161d783c42a3155b47969f0ac84a1878213c4638df85c873fc0f46594R130), but I continue to get error that no QueryClient is set if I remove it.

Reply:
I have a strong suspicion that it's because it doesn't know what URL to do the query from. I think it's because StoplightProject takes in the platform URL and has an extra `<DevPortalProvider platformUrl={platformUrl}>` and I suspect we need to do the same with the search component.

Reply:
So I did also try that and it did fix this issue. I can give it another shot, though!

Reply:
HMMM! Fascinating. OK. 🤔